pkgs/servers/jellyfin/web.nix

```diff
@@ -1,41 +1,44 @@
-{
-  lib,
-  stdenv,
-  overrideSDK,
-  fetchFromGitHub,
-  buildNpmPackage,
-  nix-update-script,
-  pkg-config,
-  xcbuild,
-  pango,
-  giflib,
-  darwin,
+{ lib
+, stdenv
+, overrideSDK
+, fetchFromGitHub
+, buildNpmPackage
+, jellyfin
+, nix-update-script
+, pkg-config
+, xcbuild
+, pango
+, giflib
+, darwin
+,
 }:
 let
   # node-canvas builds code that requires aligned_alloc,
   # which on Darwin requires at least the 10.15 SDK
   stdenv' =
     if stdenv.isDarwin then
-      overrideSDK stdenv {
-        darwinMinVersion = "10.15";
-        darwinSdkVersion = "11.0";
-      }
+      overrideSDK stdenv
+        {
+          darwinMinVersion = "10.15";
+          darwinSdkVersion = "11.0";
+        }
     else
       stdenv;
   buildNpmPackage' = buildNpmPackage.override { stdenv = stdenv'; };
 in
 buildNpmPackage' rec {
   pname = "jellyfin-web";
-  version = "10.9.2";
+  version = "10.9.3";
 
-  src = fetchFromGitHub {
+  # nixpkgs-update: no auto update
```

Comment:
Why is that? I'm not sure that if `nixpkgs-update` will try to update it, it will fail. I think though, that it won't update a PR, because it won't understand that it should also update `jellyfin`, so that evaluation will be fixed.

Reply:
It seems to be protocol to not let `nixpkgs-update` auto-update packages that should have the same version as its accompanying package. In this case, `no auto update` should help prevent it from getting out of sync with `jellyfin`. I think this may be the best option, unless there's a way for `nixpkgs-update` to couple two packages that should be updated together in one commit.